costar_dataset/block_stacking_reader_torch.py

```diff
@@ -961,6 +1050,12 @@ def ConvertImageListToNumpy(data, format='numpy', data_format='NHWC'):
                     # x = x + tuple([rgb_images[indices]])
                     # x = x + tuple([np.array(data[self.pose_name])[indices]])
 
+                    # create vector of pose, q and dq to form the joint embedding
+                    if(self.data_features_to_extract is not None and 'image_n_vec_xyz_q_dq_gripper' in self.data_features_to_extract):
```

Comment:
This looks like it is reading the values directly, I can't see all the code very well in this UI but are the preprocessing steps necessary to normalize the joint being called?

An important call for the other training code for stuff like action classification is `encode_action_and_images` which handles ensuring the input data is in the correct format. We need make sure the encoding happens here too. An explanation of the encoding definition is in the [costar dataset paper](https://arxiv.org/abs/1810.11714), just search `encoding`, an extended definition is in the appendix. This might be something I can just do later.